docs/content/preview/api/ysql/the-sql-language/statements/dml_fetch.md

```diff
@@ -53,20 +53,23 @@ A _cursor_ represents the current position in its result set. After declaring a
 
 - `FETCH ALL` and `FETCH FORWARD ALL` fetch all the rows from the row immediately after the current position through the last row, and the cursor position is left _after_ the last row. Of course, if when `FETCH ALL` (or `FETCH FORWARD ALL`) is invoked, the current position is the last row, or _after_ the last row, then an empty result is returned and the current position is left _after_ the last row.
 
-- `FETCH BACKWARD ALL` fetches all the rows from the row immediately before the current position through the first row, and the cursor position is left _before_ the first row. Of course, if when `FETCH BACKWARD ALL` is invoked, the current position is the first row, or _before_ the first row, then an empty result is returned and the current position is left _before_ the first row.
+- `FETCH BACKWARD ALL` fetches all the rows from the row immediately before the current position through the first row, and the cursor position is left _before_ the first row. Of course, if when `FETCH BACKWARD ALL` is invoked, the current position is the first row, or _before_ the first row, then an empty result is returned and the current position is left _before_ the first row.\*
 
 - The `FETCH :n` and `FETCH FORWARD :n` variants fetch exactly _:n_ rows forwards from and including the row after the current position when this many rows are available and otherwise just as many as there are to fetch analogously to how `FETCH FORWARD ALL` behaves.
-- The `FETCH BACKWARD :n` variant fetches exactly _:n_ rows backwards from and including the row before the current position when this many rows are available and otherwise just as many as there are to fetch analogously to how `FETCH BACKWARD ALL` behaves.
 
-- The `FETCH ABSOLUTE :n` variant fetches the single row at exactly the indicated absolute position. The `FETCH RELATIVE :n` variant fetches the single row at exactly the indicated relative position (_:n_ can be negative) to the current row. For both `FETCH ABSOLUTE :n` and `FETCH RELATIVE :n`, the requested row might lie before the first row or after the last row. The outcome here is the same as it is when executing other `FETCH` variants cause the current position to fall outside the range from the first through the last row in the cursor's result set. Notice that _:n_ can be negative for both the `ABSOLUTE` and the `RELATIVE` variants.
+- The `FETCH BACKWARD :n` variant fetches exactly _:n_ rows backwards from and including the row before the current position when this many rows are available and otherwise just as many as there are to fetch analogously to how `FETCH BACKWARD ALL` behaves\*.
```

Comment:
```suggestion
- The `FETCH BACKWARD :n` variant fetches exactly _:n_ rows backwards from and including the row before the current position when this many rows are available and otherwise just as many as there are to fetch analogously to how `FETCH BACKWARD ALL` behaves.\*
```

Reply:
Good catch. Will fix.